disnake/i18n.py

```diff
@@ -225,6 +225,13 @@ def _link(self, store: LocalizationProtocol) -> None:
         if self._key is not None:
             self._data = store.get(self._key)
 
+    def _copy(self) -> LocalizationValue:
+        cls = self.__class__
+        ins = cls.__new__(cls)
+        ins._key = self._key
+        ins._data = self._data.copy() if isinstance(self._data, dict) else self._data
```

Comment:
Hmm, I don't think we should copy this, as the reference returned by `LocalizationProtocol.get` is retained everywhere else. A shallow copy is fine here

The fact that modifying localization data works after everything was initialized is undocumented, but could still be useful to some.

Reply:
Changed this to shallow copy: https://github.com/DisnakeDev/disnake/pull/946/commits/2abd525839c05f7928150fab548a9ef5be8d15f8